src/test/resources/net/sf/jabref/importer/fileformat/MedlinePlainImporterTestDOI.bib

```diff
@@ -1,3 +1,3 @@
 @TechReport{,
-  DOI                      = {10.1016/j.cpr.2005.02.002}
+  article-id                      = {doi:10.1016/j.cpr.2005.02.002}
```

Comment:
Actually the DOI should be extracted by the importer to a correct format like I pointed out.